### DISCUSSION TITLE
Query equivalent of SELECT WHERE ... IN ()

### DISCUSSION THREAD

**Discussion by Nikubik** [prisma/prisma#8768] [Q&A] [answered]
Hello, 

I can't find a way to get the zip_code of a city, with a my Model like this :

```prisma
model zip_code {
  id       BigInt    @id @default(autoincrement())
  zip_code String?   @db.VarChar(5)
  city_id  BigInt
  city     city      @relation(fields: [city_id], references: [id])
  address  address[]

  @@index([city_id], name: "idx_16504_fk_zip_code_city1_idx")
}

model city {
  id         BigInt     @id @default(autoincrement())
  city_name  String?    @db.VarChar(80)
  country_id BigInt
  country    country    @relation(fields: [country_id], references: [id])
  zip_code   zip_code[]

  @@unique([city_name, country_id], name: "uc_city_name_country_id")
  @@index([country_id], name: "idx_16438_fk_city_country1_idx")
}
```

For now i only got an Object with this query:

```ts
await prisma.city.findMany({
        include: {
            country: {
                select: {
                 country_name: true
                }
            },
            zip_code: true
        }
    })
```

Result : 

``` 
{
    id: 4n,
    city_name: 'Bayeux',
    country_id: 2n,
    country: { country_name: 'France' },
    zip_code: [ [Object] ]
  },
  {
    id: 5n,
    city_name: 'Bayeux',
    country_id: 3n,
    country: { country_name: 'Suisse' },
    zip_code: [ [Object] ]
  },
  {
    id: 6n,
    city_name: 'Caen',
    country_id: 2n,
    country: { country_name: 'France' },
    zip_code: []
  },
  {
    id: 7n,
    city_name: 'Bruxelles',
    country_id: 4n,
    country: { country_name: 'Belgique' },
    zip_code: []
  }
```

Any idea how to get city_name and zip_code number ?

**Comment by ryands17** [comment, marked as answer]
@Nikubik 👋 

You already are getting `city_name` and `zip_code`. Instead of logging the return value directly, stringify it so that it's visible:

```ts
const data = await prisma.city.findMany({
        include: {
            country: {
                select: {
                 country_name: true
                }
            },
            zip_code: true
        }
})

console.log(JSON.stringify(data, null, 2))
```

**Reply by Nikubik** [reply]
Ah yes, thank you.
But now i have a problem with the id i think,
it says it can't stringify a Bigint

**Reply by ryands17** [reply]
You need to add [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#use_within_json) at the top of your file to make this work.

**Reply by Nikubik** [reply]
TS2339: Property 'toJSON' does not exist on type 'BigInt'.
:(

**Reply by ryands17** [reply]
Add `// @ts-ignore` to the line above.

**Reply by Nikubik** [reply]
I've got another problem now, but it is not relevant with stringify

node:internal/modules/cjs/loader:930
  throw err;
  ^

Error: Cannot find module './index2.js'

**Reply by ryands17** [reply]
No idea about this one. What are you using Prisma with?

**Reply by Nikubik** [reply]
With a postgres db

**Reply by ryands17** [reply]
I mean are you running it on Node or with something like Next.js?

**Reply by Nikubik** [reply]
It is Node yes

**Reply by Nikubik** [reply]
Nevermind, you help me enough, this is another problem.
Maybe add the ts ignore line in your answer directly.
Thank you.

**Reply by ryands17** [reply]
Yeah. If you can share your code, then I could check it out and get back to you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
